src/test/scala/io/github/pauljamescleary/petstore/infrastructure/endpoint/AllEndpointsSpec.scala

```diff
@@ -0,0 +1,77 @@
+package io.github.pauljamescleary.petstore.infrastructure.endpoint
+
+import cats.effect._
+import io.github.pauljamescleary.petstore.PetStoreArbitraries
+import io.github.pauljamescleary.petstore.domain.authentication.SignupRequest
```

Comment:
fix imports

Reply:
The whole classpath is required here..

Reply:
After I split the package in two, I no longer need the whole classpath...interesting.

Reply:
Yea nested package declarations are neat.